sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -22,6 +25,8 @@ class platform;
 // TODO: stop using OpenCL directly, use PI.
 namespace info {
 
+using array_type = std::array<std::byte, 16>;
```

Comment:
```suggestion
using uuid_type = std::array<std::byte, 16>;
```

The array_type is way to generic for this typedef

Reply:
We need this type as info/device_traits.def couldn't take ret_type param in this 'std::array<std::byte, 16>' format, needed a typedef as a workaround. I actually tried sycl::detail namespace first, but info/device_traits.def could only use def from sycl::info.

Reply:
These seem like limitations of the tools that process ".def" files.  Can those limitations be removed?

Reply:
I've added this under sycl::detail, I might have missed something while I tried that before.

Reply:
Nice!